### PR TITLE
Disable AdvAgg

### PR DESCRIPTION
**Pull request by doxigo**
We usually have AdvAgg enabled on prod. would be nice to turn it off on local

### CODE REVIEW

**Review by esolitos — APPROVED** [review]

**Comment by doxigo** [comment]
@esolitos am not in the organization it seems, can't merge, also could you add me to Ramsalt?

**Review by szeidler — APPROVED** [review]

**Comment by esolitos** [comment]
@doxigo Done, but I was also waiting for @szeidler review before merging. :)

**Pull request merged**
